connector_magento_product_by_sku/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Hibou Corp.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{
+ 'name': 'Magento Connector - Product by SKU',
+ 'version': '10.0.1.0.0',
+ 'author': 'Hibou,Odoo Community Association (OCA)',
+ 'license': 'AGPL-3',
+ 'category': 'Hidden',
```

Comment:
You could put the category `Connector`, which is created by the `connector` addon.